docs/metadata.md

```diff
@@ -1277,116 +1275,20 @@ ___
         STG_ORDER_PRODUCT: "LOAD_DATETIME"
     {%- endset -%}
 
-    {{ dbtvault.bridge(source_model=source_model, src_pk=src_pk,
-                       bridge_walk=bridge_walk,
-                       as_of_dates_table=as_of_dates_table,
-                       stage_tables=stage_tables,src_ldts=src_ldts) }}
-    ```
-
-=== "dbt_project.yml"
```

Comment:
We no longer use vars in a dbt_project.yml Please apply the style guide/template for tutorials here.

Reply:
Resolved